classes/YARPP_Cache.php

```diff
@@ -170,7 +170,8 @@ protected function sql($reference_ID = false, $args = array()) {
             'require_tax',
             'exclude',
             'recent',
-            'limit'
+            'limit',
+            'post__not_in',
```

Comment:
added new query param `post__not_in` so specific posts can be excluded. This was just a modification I was wanting.